### PR TITLE
Update classes.md

### PR DESCRIPTION
**Pull request by ethan-master-coding**
### Description

Added that multiple classes can be assigned to a single element.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/html/concepts/classes/classes.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Classes are HTML attributes used to select one or more elements for CSS styling
 
 ## For Styling
 
-The `class` attribute is commonly used for assigning styles to one or more elements. Inside the stylesheet, a class name is selected with the following syntax:
+The `class` attribute is commonly used for assigning styles to one or more elements. Multiple classes can be assigned to 1 element by putting a space ` ` in between the classes being assigned. Inside the stylesheet, a class name is selected with the following syntax:
```
Comment (Dusch4593) [inline content/html/concepts/classes/classes.md:20]:
```suggestion
The `class` attribute is commonly used for assigning styles to one or more elements. Multiple classes can be assigned to a single element in a space-separated list. (e.g. `class="class1 class2 class3"`)

Inside the stylesheet, a class name is selected with the following syntax:
```

**Comment by Dusch4593** [comment]
@ethan-master-coding Just took a quick look 😄

**Comment by ethan-master-coding** [comment]
@Dusch4593 All edits look good. I accepted the changes.

**Review by Dusch4593 — APPROVED** [review]
LGTM! 👍

**Pull request merged**